tests/unit/test_sde.py

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2021 The Markovflow Contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Module containing the unit tests for the SDE related functions and classes"""
+import pytest
+
+import tensorflow as tf
+import numpy as np
+import gpflow
+
+from markovflow.sde.sde_utils import euler_maruyama, linearize_sde
+from markovflow.sde import OrnsteinUhlenbeckSDE
+
+tf.random.set_seed(33)
+
+DTYPE = tf.float32
+gpflow.config.set_default_float(DTYPE)
+
+
+@pytest.fixture(name="setup")
+def _setup(state_dim, batch_shape):
+    """"""
+    t0 = 0.
+    t1 = 10.
+    n_transitions = 10
```

Comment:
That makes a convenient dt=1
You actually need to change the linearization depending on dt